cmd/crowdsec-cli/main.go

```diff
@@ -135,6 +135,8 @@ API interaction:
 	rootCmd.AddCommand(NewMetricsCmd())
 	rootCmd.AddCommand(NewDashboardCmd())
 	rootCmd.AddCommand(NewDecisionsCmd())
+	rootCmd.AddCommand(NewAlertsCmd())
+	//	rootCmd.AddCommand(NewInspectCmd())
```

Comment:
this command isn't in the revamp plan I think, can be deleted. (the command func must be kept to use it for the inspect inside other scopes)

Reply:
oohh, you are right, I just added the comment to make it build, because the code actually lacked the `NewInspectCmd()` var